### PR TITLE
Added Support for >>, <<, >>= and <<= Tokens

### LINKED ISSUE #54: Add `Assignment Shift Left`  Token at translator frontend
Implement the ASSIGN_SHIFT_LEFT token to recognize the <<= operator for performing left bitwise shift assignments.

### LINKED ISSUE #55: Add  `Assignment Shift Right`  Tokentranslator frontend
Implement the ASSIGN_SHIFT_RIGHT token to recognize the >>= operator for performing right bitwise shift assignments.

### PR DESCRIPTION
**Pull request by dotdot0**
### PR Description
Added Support for >>, <<, >>= and <<= Tokens

### Related Issue
Solves:
Closes #54 
Closes #55 

### Checklist
- [x] Have you added the necessary tests?
- [x] Only modified the files mentioned in the related issue(s)?
- [x] Are all tests passing?

### CODE REVIEW

**Comment by dotdot0** [comment]
Accidentally created a new branch from the branch of my prev PR that's why those changes are there.

**Comment by samthakur587** [comment]
hii @dotdot0 can you please resolve the merge conflicts

File: crosstl/src/translator/lexer.py [hunk 1]
```diff
@@ -33,6 +35,8 @@
     ("ASSIGN_MUL", r"\*="),
     ("ASSIGN_DIV", r"/="),
     ("WHITESPACE", r"\s+"),
+    ("SHIFT_RIGHT", r">>"),
+    ("SHIFT_LEFT", r"<<"),
     ("IF", r"\bif\b"),
```
Comment (samthakur587) [inline crosstl/src/translator/lexer.py:40]: these are bitwise left and right shift already added not the part of your task. the Assignment left and right shift is only in your task .

**Review by samthakur587 — CHANGES_REQUESTED** [review]

**Review by samthakur587 — APPROVED** [review]
hii @dotdot0 great work everything LGTM 🚀

**Pull request merged**